app.controller/app.controller.js

```diff
@@ -15,7 +19,19 @@ exports.getEndpointsRequest = (request, response, next) => {
     .then((dataFromFile) => {
       return response.status(200).send(dataFromFile);
     })
-    .then((error) => {
+    .catch((error) => {
+      next(error);
+    });
+};
+
+exports.getArticleByIdRequest = (request, response, next) => {
+  return getArticleById(request.params.article_id)
```

Comment:
This should be `fetch` or `select` articleById. At the moment it's too similar to your controller function name and is confusing.